### ISSUE TITLE
Consider merging these terms (from Pablo via email)

### ISSUE THREAD

**Issue by matentzn** [obophenotype/human-phenotype-ontology#10024]
```
HP:0001880                                        Eosinophilia
HP:0032061                                        Hypereosinophilia
```

Pablo: I do not see differences in “eosinophilia” and “hypereosinophilia” because both mean increased eosinophils…….

 
```
HP:0005984                                        Elevated maternal serum alpha-fetoprotein
HP:0011432                                        High maternal serum alpha-fetoprotein
```
 
```
HP:0008798                                        Widened sacrosciatic notch
HP:0010457                                        Widening of the sacrosciatic notch
```
 
```
HP:0025434                                        Reduced hemolytic complement activity
HP:0031906                                        Decreased total hemolytic complement activity
```
 
```
HP:0032461                                        Tiger-tail banding
HP:0045055                                        Tiger tail banding
```

**Comment by pnrobinson** [comment]
I think that in some contexts hypereosinophilia refers to severe eosinophilia. I will revise the definitions and provide a PMID

**Comment by pnrobinson** [comment]
HP:0001880  Eosinophilia
HP:0032061  Hypereosinophilia

The term “Hypereosinophilia is used with a special meaning. I have added this comment. See also PMID:20538328
Typically eosinophils in the blood are less than 0.5 billion cells per liter, corresponding to less than 5 percent of leukocytes. This term refers to a marked degree of eosinophilia with more than 1.5 billion cells per liter.

We changed the following

1) 
HP:0005984. Elevated maternal serum alpha-fetoprotein
HP:0011432  High maternal serum alpha-fetoprotein

Action: We merged these terms. I renamed the term that we keep as follows:
HP:0011432                                        
Elevated maternal circulating alpha-fetoprotein concentration

2) 
HP:0008798  Widened sacrosciatic notch
HP:0010457  Widening of the sacrosciatic notch

Action: We merged the terms. We keep this one
HP:0008798  Widened sacrosciatic notch


3)
HP:0025434   Reduced hemolytic complement activity
HP:0031906   Decreased total hemolytic complement activity

Action: We merged the terms. We keep this one (and adjusted its label):
HP:0031906   Reduced circulating CH50 activity

4)
HP:0032461  Tiger-tail banding
HP:0045055  Tiger tail banding

Action: We merged the terms. We keep this one
HP:0045055  Tiger tail banding
